formats/indent.ts

```diff
@@ -1,19 +1,20 @@
 import { ClassAttributor, Scope } from 'parchment';
 
 class IndentAttributor extends ClassAttributor {
-  add(node, value) {
-    if (value === '+1' || value === '-1') {
+  add(node: HTMLElement, _value: string | number) {
+    let value: number = 0;
```

Comment:
Let's rename the variables:

- `_value` -> `value`
- `value` -> `normalizedValue`